# Mitochondrion

A
**mitochondrion** (;[1] : **mitochondria**) is an
[organelle](organelle "wikilink") found in the
[cells](Cell_(biology) "wikilink") of most
[eukaryotes](eukaryote "wikilink"), such as
[animals](animal "wikilink"), [plants](plant "wikilink") and
[fungi](Fungus "wikilink"). Mitochondria have a double
[membrane](lipid_bilayer "wikilink") structure and use [aerobic
respiration](aerobic_respiration "wikilink") to generate [adenosine
triphosphate](adenosine_triphosphate "wikilink") (ATP), which is used
throughout the cell as a source of [chemical
energy](chemical_energy "wikilink").[2] They were discovered by [Albert
von Kölliker](Albert_von_Kölliker "wikilink") in 1857[3] in the
voluntary muscles of insects. The term *mitochondrion* was coined by
[Carl Benda](Carl_Benda "wikilink") in 1898. The mitochondrion is
popularly nicknamed the "powerhouse of the cell", a phrase coined by
[Philip Siekevitz](Philip_Siekevitz "wikilink") in a 1957 article of the
same name.[4]

Some cells in some [multicellular
organisms](multicellular_organism "wikilink") lack mitochondria (for
example, mature mammalian [red blood cells](red_blood_cell "wikilink")).
The multicellular [animal](animal "wikilink") *[Henneguya
salminicola](Henneguya_zschokkei "wikilink")* is known to have retained
mitochondrion-related organelles in association with a complete loss of
their mitochondrial genome.[5][6][7] A large number of [unicellular
organisms](unicellular_organism "wikilink"), such as
[microsporidia](microsporidia "wikilink"),
[parabasalids](parabasalid "wikilink") and
[diplomonads](diplomonad "wikilink"), have reduced or transformed their
mitochondria into other structures,[8] e.g.
[hydrogenosomes](Hydrogenosome "wikilink") and
[mitosomes](Mitosome "wikilink").[9] The
[oxymonads](oxymonad "wikilink")
*[Monocercomonoides](Monocercomonoides "wikilink")*,
*[Streblomastix](Streblomastix "wikilink")*, and *Blattamonas* have
completely lost their mitochondria.[10][11]

Mitochondria are commonly between 0.75 and 3 [μm](micrometre "wikilink")
in cross section, [12] but vary considerably in size and structure.
Unless specifically [stained](Staining "wikilink"), they are not
visible. In addition to supplying cellular energy, mitochondria are
involved in other tasks, such as [signaling](cell_signaling "wikilink"),
[cellular differentiation](cellular_differentiation "wikilink"), and
[cell death](apoptosis "wikilink"), as well as maintaining control of
the [cell cycle](cell_cycle "wikilink") and [cell
growth](cell_growth "wikilink").[13] [Mitochondrial
biogenesis](Mitochondrial_biogenesis "wikilink") is in turn temporally
coordinated with these cellular processes.[14][15] Mitochondria have
been implicated in several human disorders and conditions, such as
[mitochondrial diseases](mitochondrial_disease "wikilink"),[16] [cardiac
dysfunction](Heart_failure "wikilink"),[17] heart failure[18] and
[autism](autism "wikilink").[19]

The number of mitochondria in a cell can vary widely by
[organism](organism "wikilink"), [tissue](Tissue_(biology) "wikilink"),
and cell type. A mature red blood cell has no mitochondria,[20] whereas
a [liver cell](hepatocyte "wikilink") can have more than 2000.[21][22]
The mitochondrion is composed of compartments that carry out specialized
functions. These compartments or regions include the outer membrane,
[intermembrane space](intermembrane_space "wikilink"), [inner
membrane](inner_mitochondrial_membrane "wikilink"),
[cristae](crista "wikilink"), and
[matrix](mitochondrial_matrix "wikilink").

Although most of a eukaryotic cell's [DNA](DNA "wikilink") is contained
in the [cell nucleus](cell_nucleus "wikilink"), the mitochondrion has
[its own genome](mitochondrial_DNA "wikilink") ("mitogenome") that is
substantially similar to [bacterial](bacteria "wikilink") genomes.[23]
This finding has led to general acceptance of the [endosymbiotic
hypothesis](Symbiogenesis "wikilink") - that free-living prokaryotic
ancestors of modern mitochondria permanently fused with eukaryotic cells
in the distant past, evolving such that modern animals, plants, fungi,
and other eukaryotes are able to
[respire](Cellular_respiration "wikilink") to generate [cellular
energy](Adenosine_triphosphate "wikilink").[24]

## Structure

Mitochondria may have a number of different shapes.[25] A mitochondrion
contains outer and inner membranes composed of [phospholipid
bilayers](phospholipid_bilayer "wikilink") and
[proteins](protein "wikilink").[26] The two membranes have different
properties. Because of this double-membraned organization, there are
five distinct parts to a mitochondrion:

1.  The outer mitochondrial membrane,
2.  The intermembrane space (the space between the outer and inner
    membranes),
3.  The inner mitochondrial membrane,
4.  The [cristae](crista "wikilink") space (formed by infoldings of the
    inner membrane), and
5.  The [matrix](mitochondrial_matrix "wikilink") (space within the
    inner membrane), which is a fluid.

Mitochondria have folding to increase surface area, which in turn
increases ATP (adenosine triphosphate) production. Mitochondria stripped
of their outer membrane are called [mitoplasts](mitoplast "wikilink").

### Outer membrane

The **outer mitochondrial membrane**, which encloses the entire
organelle, is 60 to 75 [angstroms](angstrom "wikilink") (Å) thick. It
has a protein-to-phospholipid ratio similar to that of the [cell
membrane](cell_membrane "wikilink") (about 1:1 by weight). It contains
large numbers of [integral membrane
proteins](integral_membrane_protein "wikilink") called
[porins](porin_(protein) "wikilink"). A major trafficking protein is the
pore-forming [voltage-dependent anion
channel](voltage-dependent_anion_channel "wikilink") (VDAC). The
[VDAC](Voltage-dependent_anion_channel#Biological_function "wikilink")
is the primary transporter of [nucleotides](nucleotide "wikilink"),
[ions](ion "wikilink") and [metabolites](metabolite "wikilink") between
the [cytosol](cytosol "wikilink") and the intermembrane space.[27][28]
It is formed as a [beta barrel](beta_barrel "wikilink") that spans the
outer membrane, similar to that in the
[gram-negative](gram-negative "wikilink") [bacterial outer
membrane](bacterial_outer_membrane "wikilink").[29] Larger proteins can
enter the mitochondrion if a signaling sequence at their
[N-terminus](N-terminus "wikilink") binds to a large multisubunit
[protein](Mitochondrial_membrane_transport_protein "wikilink") called
[translocase in the outer
membrane](Translocase_of_the_outer_membrane "wikilink"), which then
[actively moves](active_transport "wikilink") them across the
membrane.[30] Mitochondrial [pro-proteins](protein_precursor "wikilink")
are imported through specialised translocation complexes.

The outer membrane also contains [enzymes](enzyme "wikilink") involved
in such diverse activities as the elongation of [fatty
acids](fatty_acid "wikilink"), [oxidation](oxidation "wikilink") of
[epinephrine](epinephrine "wikilink"), and the
[degradation](Biodegradation "wikilink") of
[tryptophan](tryptophan "wikilink"). These enzymes include [monoamine
oxidase](monoamine_oxidase "wikilink"),
[rotenone](rotenone "wikilink")-insensitive NADH-cytochrome c-reductase,
[kynurenine](kynurenine "wikilink")
[hydroxylase](hydroxylase "wikilink") and fatty acid Co-A
[ligase](ligase "wikilink"). Disruption of the outer membrane permits
proteins in the intermembrane space to leak into the cytosol, leading to
cell death.[31] The outer mitochondrial membrane can associate with the
endoplasmic reticulum (ER) membrane, in a structure called MAM
(mitochondria-associated ER-membrane). This is important in the
ER-mitochondria calcium signaling and is involved in the transfer of
lipids between the ER and mitochondria.[32] Outside the outer membrane
are small (diameter: 60 Å) particles named sub-units of Parson.

### Intermembrane space

The **mitochondrial intermembrane space** is the space between the outer
membrane and the inner membrane. It is also known as perimitochondrial
space. Because the outer membrane is freely permeable to small
molecules, the concentrations of small molecules, such as ions and
sugars, in the intermembrane space is the same as in the
[cytosol](cytosol "wikilink").[33] However, large proteins must have a
specific signaling sequence to be transported across the outer membrane,
so the protein composition of this space is different from the protein
composition of the [cytosol](cytosol "wikilink"). One
[protein](protein "wikilink") that is localized to the intermembrane
space in this way is [cytochrome c](cytochrome_c "wikilink").[34]

### Inner membrane

The inner mitochondrial membrane contains proteins with three types of
functions:[35]

1.  Those that perform the [electron transport
    chain](electron_transport_chain "wikilink")
    [redox](redox "wikilink") reactions
2.  [ATP synthase](ATP_synthase "wikilink"), which generates
    [ATP](Adenosine_triphosphate "wikilink") in the matrix
3.  Specific [transport proteins](membrane_transport_protein "wikilink")
    that regulate [metabolite](metabolite "wikilink") passage into and
    out of the [mitochondrial matrix](mitochondrial_matrix "wikilink")

It contains more than 151 different
[polypeptides](polypeptide "wikilink"), and has a very high
protein-to-phospholipid ratio (more than 3:1 by weight, which is about
1 protein for 15 phospholipids). The inner membrane is home to around
1/5 of the total protein in a mitochondrion.[36] Additionally, the inner
membrane is rich in an unusual phospholipid,
[cardiolipin](cardiolipin "wikilink"). This phospholipid was originally
discovered in [cow](Bos_taurus "wikilink") hearts in 1942, and is
usually characteristic of mitochondrial and bacterial plasma
membranes.[37] Cardiolipin contains four fatty acids rather than two,
and may help to make the inner membrane impermeable,[38] and its
disruption can lead to multiple clinical disorders including
neurological disorders and cancer.[39] Unlike the outer membrane, the
inner membrane does not contain porins, and is highly impermeable to all
molecules. Almost all ions and molecules require special membrane
transporters to enter or exit the matrix. Proteins are ferried into the
matrix via the [translocase of the inner
membrane](translocase_of_the_inner_membrane "wikilink") (TIM) complex or
via [OXA1L](OXA1L "wikilink").[40] In addition, there is a membrane
potential across the inner membrane, formed by the action of the
[enzymes](enzyme "wikilink") of the [electron transport
chain](electron_transport_chain "wikilink"). Inner membrane
[fusion](mitochondrial_fusion "wikilink") is mediated by the inner
membrane protein [OPA1](OPA1 "wikilink").[41]

#### Cristae

The inner mitochondrial membrane is compartmentalized into numerous
folds called [cristae](crista "wikilink"), which expand the surface area
of the inner mitochondrial membrane, enhancing its ability to produce
ATP. For typical liver mitochondria, the area of the inner membrane is
about five times as large as that of the outer membrane. This ratio is
variable and mitochondria from cells that have a greater demand for ATP,
such as muscle cells, contain even more cristae. Mitochondria within the
same cell can have substantially different crista-density, with the ones
that are required to produce more energy having much more
crista-membrane surface.[42] These folds are studded with small round
bodies known as [F particles](F-ATPase "wikilink") or oxysomes.[43]

### Matrix

The matrix is the space enclosed by the inner membrane. It contains
about 2/3 of the total proteins in a mitochondrion.[44] The matrix is
important in the production of ATP with the aid of the ATP synthase
contained in the inner membrane. The matrix contains a highly
concentrated mixture of hundreds of enzymes, special mitochondrial
[ribosomes](ribosomes "wikilink"), [tRNA](tRNA "wikilink"), and several
copies of the [mitochondrial DNA](mitochondrial_DNA "wikilink")
[genome](genome "wikilink"). Of the enzymes, the major functions include
oxidation of [pyruvate](pyruvate "wikilink") and [fatty
acids](fatty_acids "wikilink"), and the [citric acid
cycle](citric_acid_cycle "wikilink").[45] The DNA molecules are packaged
into nucleoids by proteins, one of which is [TFAM](TFAM "wikilink").[46]

## Function

The most prominent roles of mitochondria are to produce the energy
currency of the cell, [ATP](Adenosine_triphosphate "wikilink") (i.e.,
phosphorylation of [ADP](Adenosine_diphosphate "wikilink")), through
respiration and to regulate cellular
[metabolism](metabolism "wikilink").[47] The central set of reactions
involved in ATP production are collectively known as the [citric acid
cycle](citric_acid_cycle "wikilink"), or the
[Krebs](Hans_Adolf_Krebs "wikilink") cycle, and [oxidative
phosphorylation](oxidative_phosphorylation "wikilink"). However, the
mitochondrion has many other functions in addition to the production of
ATP.

### Energy conversion

A dominant role for the mitochondria is the production of ATP, as
reflected by the large number of proteins in the inner membrane for this
task. This is done by oxidizing the major products of
[glucose](glucose "wikilink"): [pyruvate](pyruvate "wikilink"), and
[NADH](NADH "wikilink"), which are produced in the cytosol.[48] This
type of [cellular respiration](cellular_respiration "wikilink"), known
as [aerobic respiration](aerobic_respiration "wikilink"), is dependent
on the presence of [oxygen](oxygen "wikilink"). When oxygen is limited,
the glycolytic products will be metabolized by [anaerobic
fermentation](Fermentation_(biochemistry) "wikilink"), a process that is
independent of the mitochondria.[49] The production of ATP from glucose
and oxygen has an approximately 13-times higher yield during aerobic
respiration compared to fermentation.[50] Plant mitochondria can also
produce a limited amount of ATP either by breaking the sugar produced
during photosynthesis or without oxygen by using the alternate substrate
[nitrite](nitrite "wikilink").[51] ATP crosses out through the inner
membrane with the help of a [specific
protein](ATP–ADP_translocase "wikilink"), and across the outer membrane
via [porins](Porin_(protein) "wikilink").[52] After conversion of ATP to
[ADP](Adenosine_diphosphate "wikilink") by
[dephosphorylation](dephosphorylation "wikilink") that releases energy,
ADP returns via the same route.

#### Pyruvate and the citric acid cycle

[Pyruvate](Pyruvate "wikilink") molecules produced by
[glycolysis](glycolysis "wikilink") are [actively
transported](active_transport "wikilink") across the inner mitochondrial
membrane, and into the matrix where they can either be
[oxidized](Redox "wikilink") and combined with [coenzyme
A](coenzyme_A "wikilink") to form CO,
[acetyl-CoA](acetyl-CoA "wikilink"), and [NADH](NADH "wikilink"),[53] or
they can be [carboxylated](carboxylated "wikilink") (by [pyruvate
carboxylase](pyruvate_carboxylase "wikilink")) to form oxaloacetate.
This latter reaction "fills up" the amount of oxaloacetate in the citric
acid cycle and is therefore an [anaplerotic
reaction](anaplerotic_reaction "wikilink"), increasing the cycle's
capacity to metabolize acetyl-CoA when the tissue's energy needs (e.g.,
in [muscle](striated_muscle_tissue "wikilink")) are suddenly increased
by activity.[54]

In the citric acid cycle, all the intermediates (e.g.
[citrate](citrate "wikilink"), [iso-citrate](Isocitric_acid "wikilink"),
[alpha-ketoglutarate](Alpha-Ketoglutaric_acid "wikilink"), succinate,
[fumarate](Fumaric_acid "wikilink"), [malate](Malic_acid "wikilink") and
oxaloacetate) are regenerated during each turn of the cycle. Adding more
of any of these intermediates to the mitochondrion therefore means that
the additional amount is retained within the cycle, increasing all the
other intermediates as one is converted into the other. Hence, the
addition of any one of them to the cycle has an
[anaplerotic](Anaplerotic_reactions "wikilink") effect, and its removal
has a cataplerotic effect. These anaplerotic and
[cataplerotic](cataplerotic "wikilink") reactions will, during the
course of the cycle, increase or decrease the amount of oxaloacetate
available to combine with acetyl-CoA to form citric acid. This in turn
increases or decreases the rate of
[ATP](Adenosine_triphosphate "wikilink") production by the
mitochondrion, and thus the availability of ATP to the cell.[55]

Acetyl-CoA, on the other hand, derived from pyruvate oxidation, or from
the [beta-oxidation](beta-oxidation "wikilink") of [fatty
acids](fatty_acids "wikilink"), is the only fuel to enter the citric
acid cycle. With each turn of the cycle one molecule of acetyl-CoA is
consumed for every molecule of oxaloacetate present in the mitochondrial
matrix, and is never regenerated. It is the oxidation of the acetate
portion of acetyl-CoA that produces CO and water, with the energy thus
released captured in the form of ATP.[56]

In the liver, the [carboxylation](carboxylation "wikilink") of
[cytosolic](cytosol "wikilink") pyruvate into intra-mitochondrial
oxaloacetate is an early step in the
[gluconeogenic](gluconeogenesis "wikilink") pathway, which converts
[lactate](lactic_acid "wikilink") and de-aminated
[alanine](alanine "wikilink") into glucose,[57][58] under the influence
of high levels of [glucagon](glucagon "wikilink") and/or
[epinephrine](epinephrine "wikilink") in the blood.[59] Here, the
addition of oxaloacetate to the mitochondrion does not have a net
anaplerotic effect, as another citric acid cycle intermediate (malate)
is immediately removed from the mitochondrion to be converted to
cytosolic oxaloacetate, and ultimately to glucose, in a process that is
almost the reverse of [glycolysis](glycolysis "wikilink").[60]

The enzymes of the citric acid cycle are located in the mitochondrial
matrix, with the exception of [succinate
dehydrogenase](succinate_dehydrogenase "wikilink"), which is bound to
the inner mitochondrial membrane as part of Complex II.[61] The citric
acid cycle oxidizes the acetyl-CoA to carbon dioxide, and, in the
process, produces reduced cofactors (three molecules of
[NADH](NADH "wikilink") and one molecule of [FADH](FADH2 "wikilink"))
that are a source of electrons for the [electron transport
chain](electron_transport_chain "wikilink"), and a molecule of
[GTP](Guanosine_triphosphate "wikilink") (which is readily converted to
an ATP).[62]

#### O2 and NADH: energy-releasing reactions

The electrons from NADH and FADH are transferred to oxygen (O) and
hydrogen (protons) in several steps via an electron transport chain.
NADH and FADH molecules are produced within the matrix via the citric
acid cycle and in the cytoplasm by [glycolysis](glycolysis "wikilink").
[Reducing equivalents](Reducing_equivalent "wikilink") from the
cytoplasm can be imported via the [malate-aspartate
shuttle](malate-aspartate_shuttle "wikilink") system of
[antiporter](antiporter "wikilink") proteins or fed into the electron
transport chain using a [glycerol phosphate
shuttle](glycerol_phosphate_shuttle "wikilink").[63]

The major energy-releasing reactions [64][65] that make the
mitochondrion the "powerhouse of the cell" occur at [protein complexes
I, III and
IV](Electron_transport_chain#Mitochondrial_redox_carriers "wikilink") in
the inner mitochondrial membrane ([NADH dehydrogenase
(ubiquinone)](NADH_dehydrogenase_(ubiquinone) "wikilink"), [cytochrome c
reductase](Coenzyme_Q_-_cytochrome_c_reductase "wikilink"), and
[cytochrome c oxidase](cytochrome_c_oxidase "wikilink")). At [complex
IV](complex_IV "wikilink"), O<sub>2</sub> reacts with the reduced form
of iron in [cytochrome c](cytochrome_c "wikilink"):

<chem>O2{} + 4H+(aq){} + 4 Fe^{2+}(cyt\\c) -&gt; 2H2O{} + 4
Fe^{3+}(cyt\\c) </chem>
*Δ*<sub>*r*</sub>*G*<sup>*o*′</sup> = −218 kJ/mol

releasing a lot of [free energy](Gibbs_free_energy "wikilink")[66][67]
from the reactants without breaking bonds of an organic fuel. The free
energy put in to remove an electron from Fe<sup>2+</sup> is released at
[complex III](complex_III "wikilink") when Fe<sup>3+</sup> of cytochrome
c reacts to oxidize [ubiquinol](ubiquinol "wikilink") (QH<sub>2</sub>):

<chem>2 Fe^{3+}(cyt\\c){} + QH2 -&gt; 2 Fe^{2+}(cyt\\c){} + Q{} +
2H+(aq)</chem> *Δ*<sub>*r*</sub>*G*<sup>*o*′</sup> = −30 kJ/mol

The [ubiquinone](ubiquinone "wikilink") (Q) generated reacts, in
[complex I](complex_I "wikilink"), with NADH:

<chem>Q + H+(aq){} + NADH -&gt; QH2 + NAD+ { }</chem>
*Δ*<sub>*r*</sub>*G*<sup>*o*′</sup> = −81 kJ/mol

While the reactions are controlled by an electron transport chain, free
electrons are not amongst the reactants or products in the three
reactions shown and therefore do not affect the free energy released,
which is used to pump [protons](Hydrogen_ion "wikilink") (H) into the
intermembrane space. This process is efficient, but a small percentage
of electrons may prematurely reduce oxygen, forming [reactive oxygen
species](reactive_oxygen_species "wikilink") such as
[superoxide](superoxide "wikilink").[68] This can cause [oxidative
stress](oxidative_stress "wikilink") in the mitochondria and may
contribute to the decline in mitochondrial function associated with
aging.[69]

As the proton concentration increases in the intermembrane space, a
strong [electrochemical gradient](electrochemical_gradient "wikilink")
is established across the inner membrane. The protons can return to the
matrix through the [ATP synthase](ATP_synthase "wikilink") complex, and
their potential energy is used to synthesize
[ATP](Adenosine_triphosphate "wikilink") from ADP and inorganic
phosphate (P).[70] This process is called
[chemiosmosis](chemiosmosis "wikilink"), and was first described by
[Peter Mitchell](Peter_D._Mitchell "wikilink"),[71][72] who was awarded
the 1978 [Nobel Prize in Chemistry](Nobel_Prize_in_Chemistry "wikilink")
for his work. Later, part of the 1997 Nobel Prize in Chemistry was
awarded to [Paul D. Boyer](Paul_D._Boyer "wikilink") and [John E.
Walker](John_E._Walker "wikilink") for their clarification of the
working mechanism of ATP synthase.[73]

#### Heat production

Under certain conditions, protons can re-enter the mitochondrial matrix
without contributing to ATP synthesis. This process is known as *proton
leak* or [mitochondrial uncoupling](mitochondrial_uncoupling "wikilink")
and is due to the [facilitated
diffusion](facilitated_diffusion "wikilink") of protons into the matrix.
The process results in the unharnessed potential energy of the proton
[electrochemical](Electrochemistry "wikilink") gradient being released
as heat.[74] The process is mediated by a proton channel called
[thermogenin](thermogenin "wikilink"), or [UCP1](UCP1 "wikilink").[75]
Thermogenin is primarily found in [brown adipose
tissue](brown_adipose_tissue "wikilink"), or brown fat, and is
responsible for non-shivering thermogenesis. Brown adipose tissue is
found in mammals, and is at its highest levels in early life and in
hibernating animals. In humans, brown adipose tissue is present at birth
and decreases with age.[76]

### Mitochondrial fatty acid synthesis

Mitochondrial fatty acid synthesis (mtFASII) is essential for cellular
respiration and mitochondrial biogenesis.[77] It is also thought to play
a role as a mediator in [intracellular
signaling](Cell_signaling "wikilink") due to its influence on the levels
of bioactive lipids, such as
[lysophospholipids](Lysophospholipase "wikilink") and
[sphingolipids](sphingolipid "wikilink").[78]

[Octanoyl-ACP](Octanoyl-(acyl-carrier_protein):protein_N-octanoyltransferase "wikilink")
(C8) is considered to be the most important end product of mtFASII,
which also forms the starting substrate of [lipoic
acid](lipoic_acid "wikilink") biosynthesis.[79] Since lipoic acid is the
cofactor of important mitochondrial enzyme complexes, such as the
[pyruvate dehydrogenase
complex](pyruvate_dehydrogenase_complex "wikilink") (PDC),
[α-ketoglutarate dehydrogenase
complex](Oxoglutarate_dehydrogenase_complex "wikilink") (OGDC),
[branched-chain α-ketoacid dehydrogenase
complex](Branched-chain_alpha-keto_acid_dehydrogenase_complex "wikilink")
(BCKDC), and in the [glycine cleavage
system](glycine_cleavage_system "wikilink") (GCS), mtFASII has an
influence on energy metabolism.[80]

Other products of mtFASII play a role in the regulation of mitochondrial
translation, [FeS cluster
biogenesis](Iron-sulfur_cluster_biosynthesis "wikilink") and assembly of
oxidative phosphorylation complexes.[81]

Furthermore, with the help of mtFASII and acylated ACP, acetyl-CoA
regulates its consumption in mitochondria.[82]

### Uptake, storage and release of calcium ions

The concentrations of free calcium in the cell can regulate an array of
reactions and is important for [signal
transduction](signal_transduction "wikilink") in the cell. Mitochondria
can transiently [store calcium](Calcium_storage "wikilink"), a
contributing process for the cell's homeostasis of calcium.[83] [84]
Their ability to rapidly take in calcium for later release makes them
good "cytosolic buffers" for calcium.[85][86][87] The endoplasmic
reticulum (ER) is the most significant storage site of calcium,[88] and
there is a significant interplay between the mitochondrion and ER with
regard to calcium.[89] The calcium is taken up into the
[matrix](mitochondrial_matrix "wikilink") by the [mitochondrial calcium
uniporter](mitochondrial_calcium_uniporter "wikilink") on the [inner
mitochondrial membrane](inner_mitochondrial_membrane "wikilink").[90] It
is primarily driven by the mitochondrial [membrane
potential](membrane_potential "wikilink").[91] Release of this calcium
back into the cell's interior can occur via a sodium-calcium exchange
protein or via "calcium-induced-calcium-release" pathways.[92] This can
initiate calcium spikes or calcium waves with large changes in the
membrane potential. These can activate a series of [second messenger
system](second_messenger_system "wikilink") proteins that can coordinate
processes such as [neurotransmitter
release](Synaptic_vesicle "wikilink") in nerve cells and release of
[hormones](hormone "wikilink") in endocrine cells.[93]

Ca influx to the mitochondrial matrix has recently been implicated as a
mechanism to regulate respiratory
[bioenergetics](bioenergetics "wikilink") by allowing the
electrochemical potential across the membrane to transiently "pulse"
from ΔΨ-dominated to pH-dominated, facilitating a reduction of
[oxidative stress](oxidative_stress "wikilink").[94] In neurons,
concomitant increases in cytosolic and mitochondrial calcium act to
synchronize neuronal activity with mitochondrial energy metabolism.
Mitochondrial matrix calcium levels can reach the tens of micromolar
levels, which is necessary for the activation of [isocitrate
dehydrogenase](isocitrate_dehydrogenase "wikilink"), one of the key
regulatory enzymes of the [Krebs cycle](Krebs_cycle "wikilink").[95]

### Cellular proliferation regulation

The relationship between cellular proliferation and mitochondria has
been investigated. Tumor cells require ample ATP to synthesize bioactive
compounds such as [lipids](lipid "wikilink"),
[proteins](protein "wikilink"), and [nucleotides](nucleotide "wikilink")
for rapid proliferation.[96] The majority of ATP in tumor cells is
generated via the [oxidative
phosphorylation](oxidative_phosphorylation "wikilink") pathway
(OxPhos).[97] Interference with OxPhos cause [cell
cycle](cell_cycle "wikilink") arrest suggesting that mitochondria play a
role in cell proliferation.[98] Mitochondrial ATP production is also
vital for [cell division](cell_division "wikilink") and differentiation
in infection [99] in addition to basic functions in the cell including
the regulation of cell volume, solute
[concentration](concentration "wikilink"), and cellular
architecture.[100][101][102] ATP levels differ at various stages of the
cell cycle suggesting that there is a relationship between the abundance
of ATP and the cell's ability to enter a new cell cycle.[103] ATP's role
in the basic functions of the cell make the [cell
cycle](cell_cycle "wikilink") sensitive to changes in the availability
of mitochondrial derived ATP.[104] The variation in ATP levels at
different stages of the cell cycle support the hypothesis that
mitochondria play an important role in cell cycle regulation.[105]
Although the specific mechanisms between mitochondria and the cell cycle
regulation is not well understood, studies have shown that low energy
cell cycle checkpoints monitor the energy capability before committing
to another round of cell division.[106]

### Additional functions

Mitochondria play a central role in many other
[metabolic](metabolism "wikilink") tasks, such as:

-   Signaling through mitochondrial [reactive oxygen
    species](reactive_oxygen_species "wikilink")[107]
-   Regulation of the [membrane
    potential](membrane_potential "wikilink")[108]
-   [Apoptosis](Apoptosis "wikilink")-programmed cell death[109]
-   Calcium signaling (including calcium-evoked apoptosis)[110]
-   Regulation of cellular [metabolism](metabolism "wikilink")[111]
-   Certain [heme](heme "wikilink") synthesis reactions[112] (see also:
    *[Porphyrin](Porphyrin "wikilink")*)
-   [Steroid](Steroid "wikilink") synthesis[113]
-   Hormonal signaling[114] – mitochondria are sensitive and responsive
    to hormones, in part by the action of mitochondrial estrogen
    receptors (mtERs). These receptors have been found in various
    tissues and cell types, including brain[115] and heart[116]
-   Immune signaling[117]
-   Neuronal mitochondria also contribute to cellular quality control by
    reporting neuronal status towards microglia through specialised
    somatic-junctions.[118]
-   Mitochondria of developing neurons contribute to intercellular
    signaling towards [microglia](microglia "wikilink"), which
    communication is indispensable for proper regulation of brain
    development.[119]

Some mitochondrial functions are performed only in specific types of
cells. For example, mitochondria in [liver cells](liver_cell "wikilink")
contain enzymes that allow them to detoxify
[ammonia](ammonia "wikilink"), a waste product of protein metabolism. A
mutation in the genes regulating any of these functions can result in
[mitochondrial diseases](mitochondrial_disease "wikilink").

Mitochondrial proteins (proteins transcribed from mitochondrial DNA)
vary depending on the tissue and the species. In humans, 615 distinct
types of proteins have been identified from [cardiac](heart "wikilink")
mitochondria,[120] whereas in [rats](Murinae "wikilink"), 940 proteins
have been reported.[121] The mitochondrial
[proteome](proteome "wikilink") is thought to be dynamically
regulated.[122]

## Organization and distribution

Mitochondria (or related structures) are found in all
[eukaryotes](eukaryote "wikilink") (except the
[Oxymonad](Oxymonad "wikilink")
*[Monocercomonoides](Monocercomonoides "wikilink")*).[123] Although
commonly depicted as bean-like structures they form a highly dynamic
network in the majority of cells where they constantly undergo
[fission](mitochondrial_fission "wikilink") and
[fusion](mitochondrial_fusion "wikilink"). The population of all the
mitochondria of a given cell constitutes the chondriome.[124]
Mitochondria vary in number and location according to cell type. A
single mitochondrion is often found in unicellular organisms, while
human liver cells have about 1000–2000 mitochondria per cell, making up
1/5 of the cell volume.[125] The mitochondrial content of otherwise
similar cells can vary substantially in size and membrane
potential,[126] with differences arising from sources including uneven
partitioning at cell division, leading to [extrinsic
differences](cellular_noise "wikilink") in ATP levels and downstream
cellular processes.[127] The mitochondria can be found nestled between
[myofibrils](myofibril "wikilink") of [muscle](muscle "wikilink") or
wrapped around the [sperm](sperm "wikilink")
[flagellum](flagellum "wikilink").[128] Often, they form a complex 3D
branching network inside the cell with the
[cytoskeleton](cytoskeleton "wikilink"). The association with the
cytoskeleton determines mitochondrial shape, which can affect the
function as well:[129] different structures of the mitochondrial network
may afford the population a variety of physical, chemical, and
signalling advantages or disadvantages.[130] Mitochondria in cells are
always distributed along microtubules and the distribution of these
organelles is also correlated with the [endoplasmic
reticulum](endoplasmic_reticulum "wikilink").[131] Recent evidence
suggests that [vimentin](vimentin "wikilink"), one of the components of
the cytoskeleton, is also critical to the association with the
cytoskeleton.[132]

### Mitochondria-associated ER membrane (MAM)

The mitochondria-associated ER membrane (MAM) is another structural
element that is increasingly recognized for its critical role in
cellular physiology and [homeostasis](homeostasis "wikilink"). Once
considered a technical snag in cell fractionation techniques, the
alleged ER vesicle contaminants that invariably appeared in the
mitochondrial fraction have been re-identified as membranous structures
derived from the MAM—the interface between mitochondria and the ER.[133]
Physical coupling between these two organelles had previously been
observed in electron micrographs and has more recently been probed with
[fluorescence microscopy](fluorescence_microscopy "wikilink").[134] Such
studies estimate that at the MAM, which may comprise up to 20% of the
mitochondrial outer membrane, the ER and mitochondria are separated by a
mere 10–25 nm and held together by protein tethering
complexes.[135][136][137]

Purified MAM from subcellular fractionation is enriched in enzymes
involved in phospholipid exchange, in addition to channels associated
with Ca signaling.[138][139] These hints of a prominent role for the MAM
in the regulation of cellular lipid stores and signal transduction have
been borne out, with significant implications for
mitochondrial-associated cellular phenomena, as discussed below. Not
only has the MAM provided insight into the mechanistic basis underlying
such physiological processes as intrinsic apoptosis and the propagation
of calcium signaling, but it also favors a more refined view of the
mitochondria. Though often seen as static, isolated 'powerhouses'
hijacked for cellular metabolism through an ancient endosymbiotic event,
the evolution of the MAM underscores the extent to which mitochondria
have been integrated into overall cellular physiology, with intimate
physical and functional coupling to the endomembrane system.

#### Phospholipid transfer

The MAM is enriched in enzymes involved in lipid biosynthesis, such as
phosphatidylserine synthase on the ER face and phosphatidylserine
decarboxylase on the mitochondrial face.[140][141] Because mitochondria
are dynamic organelles constantly undergoing
[fission](mitochondrial_fission "wikilink") and
[fusion](mitochondrial_fusion "wikilink") events, they require a
constant and well-regulated supply of phospholipids for membrane
integrity.[142][143] But mitochondria are not only a destination for the
phospholipids they finish synthesis of; rather, this organelle also
plays a role in inter-organelle trafficking of the intermediates and
products of phospholipid biosynthetic pathways, ceramide and cholesterol
metabolism, and glycosphingolipid anabolism.[144][145]

Such trafficking capacity depends on the MAM, which has been shown to
facilitate transfer of lipid intermediates between organelles.[146] In
contrast to the standard vesicular mechanism of lipid transfer, evidence
indicates that the physical proximity of the ER and mitochondrial
membranes at the MAM allows for lipid flipping between opposed
bilayers.[147] Despite this unusual and seemingly energetically
unfavorable mechanism, such transport does not require ATP.[148]
Instead, in yeast, it has been shown to be dependent on a
[multiprotein](Protein_complex "wikilink") tethering structure termed
the ER-mitochondria encounter structure, or ERMES, although it remains
unclear whether this structure directly mediates lipid transfer or is
required to keep the membranes in sufficiently close proximity to lower
the [energy barrier](activation_energy "wikilink") for
[lipid](lipid "wikilink") flipping.[149][150]

The MAM may also be part of the secretory pathway, in addition to its
role in intracellular lipid trafficking. In particular, the MAM appears
to be an intermediate destination between the rough ER and the Golgi in
the pathway that leads to [very-low-density
lipoprotein](very-low-density_lipoprotein "wikilink"), or VLDL, assembly
and secretion.[151][152] The MAM thus serves as a critical metabolic and
trafficking hub in lipid metabolism.

#### Calcium signaling

A critical role for the ER in calcium signaling was acknowledged before
such a role for the mitochondria was widely accepted, in part because
the low affinity of Ca channels localized to the outer mitochondrial
membrane seemed to contradict this organelle's purported responsiveness
to changes in intracellular Ca flux.[153][154] But the presence of the
MAM resolves this apparent contradiction: the close physical association
between the two organelles results in Ca microdomains at contact points
that facilitate efficient Ca transmission from the ER to the
mitochondria.[155] Transmission occurs in response to so-called "Ca
puffs" generated by spontaneous clustering and activation of
[IP3R](IP3R "wikilink"), a canonical ER membrane Ca channel.[156][157]

The fate of these puffs—in particular, whether they remain restricted to
isolated locales or integrated into Ca waves for propagation throughout
the cell—is determined in large part by MAM dynamics. Although reuptake
of Ca by the ER (concomitant with its release) modulates the intensity
of the puffs, thus insulating mitochondria to a certain degree from high
Ca exposure, the MAM often serves as a firewall that essentially buffers
Ca puffs by acting as a sink into which free ions released into the
cytosol can be funneled.[158][159][160] This Ca tunneling occurs through
the low-affinity Ca receptor [VDAC1](VDAC1 "wikilink"), which recently
has been shown to be physically
[tethered](tether_(cell_biology) "wikilink") to the IP3R clusters on the
ER membrane and enriched at the MAM.[161][162][163] The ability of
mitochondria to serve as a Ca sink is a result of the electrochemical
gradient generated during oxidative phosphorylation, which makes
tunneling of the cation an exergonic process.[164] Normal, mild calcium
influx from cytosol into the mitochondrial matrix causes transient
depolarization that is corrected by pumping out protons.

But transmission of Ca is not unidirectional; rather, it is a two-way
street.[165] The properties of the Ca pump SERCA and the channel IP3R
present on the ER membrane facilitate feedback regulation coordinated by
MAM function. In particular, the clearance of Ca by the MAM allows for
[spatio-temporal patterning](spatio-temporal_pattern "wikilink") of Ca
signaling because Ca alters IP3R activity in a biphasic manner.[166]
[SERCA](SERCA "wikilink") is likewise affected by mitochondrial
feedback: uptake of Ca by the MAM stimulates ATP production, thus
providing energy that enables SERCA to reload the ER with Ca for
continued Ca efflux at the MAM.[167][168] Thus, the MAM is not a passive
buffer for Ca puffs; rather it helps modulate further Ca signaling
through feedback loops that affect ER dynamics.

Regulating ER release of Ca at the MAM is especially critical because
only a certain window of Ca uptake sustains the mitochondria, and
consequently the cell, at homeostasis. Sufficient intraorganelle Ca
signaling is required to stimulate metabolism by activating
dehydrogenase enzymes critical to flux through the citric acid
cycle.[169][170] However, once Ca signaling in the mitochondria passes a
certain threshold, it stimulates the intrinsic pathway of apoptosis in
part by collapsing the mitochondrial membrane potential required for
metabolism.[171] Studies examining the role of pro- and anti-apoptotic
factors support this model; for example, the anti-apoptotic factor Bcl-2
has been shown to interact with IP3Rs to reduce Ca filling of the ER,
leading to reduced efflux at the MAM and preventing collapse of the
mitochondrial membrane potential post-apoptotic stimuli.[172] Given the
need for such fine regulation of Ca signaling, it is perhaps
unsurprising that dysregulated mitochondrial Ca has been implicated in
several neurodegenerative diseases, while the catalogue of tumor
suppressors includes a few that are enriched at the MAM.[173]

#### Molecular basis for tethering

Recent advances in the identification of the
[tethers](tether_(cell_biology) "wikilink") between the mitochondrial
and ER membranes suggest that the scaffolding function of the molecular
elements involved is secondary to other, non-structural functions. In
yeast, ERMES, a multiprotein complex of interacting ER- and
mitochondrial-resident membrane proteins, is required for lipid transfer
at the MAM and exemplifies this principle. One of its components, for
example, is also a constituent of the protein complex required for
insertion of transmembrane beta-barrel proteins into the lipid
bilayer.[174] However, a [homologue](Homology_(biology) "wikilink") of
the ERMES complex has not yet been identified in mammalian cells. Other
proteins implicated in scaffolding likewise have functions independent
of structural tethering at the MAM; for example, ER-resident and
mitochondrial-resident mitofusins form heterocomplexes that regulate the
number of inter-organelle contact sites, although mitofusins were first
identified for their role in [fission](Mitochondrial_fission "wikilink")
and [fusion](Mitochondrial_fusion "wikilink") events between individual
mitochondria.[175] [Glucose](Glucose "wikilink")-related protein 75
(grp75) is another dual-function protein. In addition to the matrix pool
of grp75, a portion serves as a chaperone that physically links the
mitochondrial and ER Ca channels VDAC and IP3R for efficient Ca
transmission at the MAM.[176][177] Another potential tether is
[Sigma-1R](Sigma-1_receptor "wikilink"), a non-opioid receptor whose
stabilization of ER-resident IP3R may preserve communication at the MAM
during the metabolic stress response.[178][179]


#### Perspective

The MAM is a critical signaling, metabolic, and trafficking hub in the
cell that allows for the integration of ER and mitochondrial physiology.
Coupling between these organelles is not simply structural but
functional as well and critical for overall cellular physiology and
[homeostasis](homeostasis "wikilink"). The MAM thus offers a perspective
on mitochondria that diverges from the traditional view of this
organelle as a static, isolated unit appropriated for its metabolic
capacity by the cell.[180] Instead, this mitochondrial-ER interface
emphasizes the integration of the mitochondria, the product of an
endosymbiotic event, into diverse cellular processes. Recently it has
also been shown, that mitochondria and MAM-s in neurons are anchored to
specialised intercellular communication sites (so called
somatic-junctions). [Microglial](Microglia "wikilink") processes monitor
and protect neuronal functions at these sites, and MAM-s are supposed to
have an important role in this type of cellular quality-control.[181]

## <span class="anchor" id="origin_and_evolution_anchor">Origin and evolution</span>

There are two hypotheses about the origin of mitochondria:
[endosymbiotic](endosymbiotic_theory "wikilink") and
[autogenous](autotransplantation "wikilink"). The endosymbiotic
hypothesis suggests that mitochondria were originally
[prokaryotic](Prokaryote "wikilink") cells, capable of implementing
oxidative mechanisms that were not possible for eukaryotic cells; they
became [endosymbionts](endosymbiont "wikilink") living inside the
eukaryote.[182][183][184][185] In the autogenous hypothesis,
mitochondria were born by splitting off a portion of DNA from the
nucleus of the eukaryotic cell at the time of divergence with the
prokaryotes; this DNA portion would have been enclosed by membranes,
which could not be crossed by proteins. Since mitochondria have many
features in common with [bacteria](bacteria "wikilink"), the
endosymbiotic hypothesis is the more widely accepted of the two
accounts.[186][187]

A [mitochondrion contains DNA](Mitochondrial_DNA "wikilink"), which is
organized as several copies of a single, usually
[circular](Mitochondrial_DNA#Circular_versus_linear "wikilink")
[chromosome](chromosome "wikilink"). This mitochondrial chromosome
contains genes for [redox](redox "wikilink") proteins, such as those of
the respiratory chain. The [CoRR hypothesis](CoRR_hypothesis "wikilink")
proposes that this co-location is required for redox regulation. The
mitochondrial [genome](genome "wikilink") codes for some
[RNAs](RNA "wikilink") of [ribosomes](ribosome "wikilink"), and the
22 [tRNAs](tRNA "wikilink") necessary for the translation of
[mRNAs](mRNA "wikilink") into protein. The circular structure is also
found in prokaryotes. The
[proto-mitochondrion](proto-mitochondrion "wikilink") was probably
closely related to *[Rickettsia](Rickettsia "wikilink")*.[188][189]
However, the exact relationship of the ancestor of mitochondria to the
[alphaproteobacteria](alphaproteobacteria "wikilink") and whether the
mitochondrion was formed at the same time or after the nucleus, remains
controversial.[190] For example, it has been suggested that the [SAR11
clade](SAR11_clade "wikilink") of bacteria shares a relatively recent
common ancestor with the mitochondria,[191] while
[phylogenomic](phylogenomic "wikilink") analyses indicate that
mitochondria evolved from a [Pseudomonadota](Pseudomonadota "wikilink")
lineage that is closely related to or a member of
[alphaproteobacteria](alphaproteobacteria "wikilink").[192][193] Some
papers describe mitochondria as sister to the alphaproteobactera,
together forming the sister the marineproteo1 group, together forming
the sister to
[Magnetococcidae](Magnetococcidae "wikilink").[194][195][196][197]

The ribosomes coded for by the mitochondrial DNA are similar to those
from bacteria in size and structure.[198] They closely resemble the
bacterial [70S](Ribosome#Structure "wikilink") ribosome and not the
[80S](Ribosome#Structure "wikilink") [cytoplasmic](cytoplasm "wikilink")
ribosomes, which are coded for by [nuclear](Cell_nucleus "wikilink")
DNA.

The [endosymbiotic](endosymbiotic "wikilink") relationship of
mitochondria with their host cells was popularized by [Lynn
Margulis](Lynn_Margulis "wikilink").[199] The [endosymbiotic
hypothesis](Endosymbiotic_theory "wikilink") suggests that mitochondria
descended from aerobic bacteria that somehow survived
[endocytosis](endocytosis "wikilink") by another cell, and became
incorporated into the [cytoplasm](cytoplasm "wikilink"). The ability of
these bacteria to conduct [respiration](Cellular_respiration "wikilink")
in host cells that had relied on [glycolysis](glycolysis "wikilink") and
[fermentation](Fermentation_(biochemistry) "wikilink") would have
provided a considerable evolutionary advantage. This symbiotic
relationship probably developed 1.7 to 2 billion years ago.[200][201]
![Evolution of
MROs](The-origins-of-mitochondrion-related-organelles-A-hypothetical-scenario-for-the.png "Evolution of MROs")
A few groups of unicellular eukaryotes have only vestigial mitochondria
or derived structures: The [microsporidians](microsporidia "wikilink"),
[metamonads](metamonad "wikilink"), and
[archamoebae](archamoebae "wikilink").[202] These groups appear as the
most primitive eukaryotes on [phylogenetic
trees](phylogenetic_trees "wikilink") constructed using
[rRNA](rRNA "wikilink") information, which once suggested that they
appeared before the origin of mitochondria. However, this is now known
to be an artifact of *[long-branch
attraction](long-branch_attraction "wikilink")*: They are derived groups
and retain genes or organelles derived from mitochondria (e. g.,
[mitosomes](mitosome "wikilink") and
[hydrogenosomes](hydrogenosome "wikilink")).[203] Hydrogenosomes,
mitosomes, and related organelles as found in some
[loricifera](loricifera "wikilink") (e. g.
*[Spinoloricus](Spinoloricus "wikilink")*)[204][205] and
[myxozoa](myxozoa "wikilink") (e. g. *[Henneguya
zschokkei](Henneguya_zschokkei "wikilink")*) are together classified as
MROs, mitochondrion-related organelles.[206][207]

[Monocercomonoides](Monocercomonoides "wikilink") and other
[oxymonads](oxymonad "wikilink") appear to have lost their mitochondria
completely and at least some of the mitochondrial functions seem to be
carried out by cytoplasmic proteins now.[208][209][210]

## Mitochondrial genetics

Mitochondria contain their own genome. The [human mitochondrial
genome](Human_mitochondrial_genetics "wikilink") is a circular
double-stranded [DNA](DNA "wikilink") molecule of about
16 [kilobases](base_pair "wikilink").[211] It encodes 37 genes: 13 for
[subunits](Protein_subunit "wikilink") of respiratory complexes I, III,
IV and V, 22 for mitochondrial [tRNA](tRNA "wikilink") (for the 20
standard amino acids, plus an extra gene for leucine and serine), and 2
for [rRNA](rRNA "wikilink") (12S and 16S rRNA).[212] One mitochondrion
can contain two to ten copies of its DNA.[213] One of the two
mitochondrial DNA (mtDNA) strands has a disproportionately higher ratio
of the heavier nucleotides adenine and guanine, and this is termed the
heavy strand (or H strand), whereas the other strand is termed the light
strand (or L strand). The weight difference allows the two strands to be
separated by [centrifugation](centrifugation "wikilink"). mtDNA has one
long non-coding stretch known as the non-coding region (NCR), which
contains the heavy strand promoter (HSP) and light strand promoter (LSP)
for RNA transcription, the origin of replication for the H strand (OriH)
localized on the L strand, three conserved sequence boxes (CSBs 1–3),
and a termination-associated sequence (TAS). The origin of replication
for the L strand (OriL) is localized on the H strand 11,000 bp
downstream of OriH, located within a cluster of genes coding for
tRNA.[214]

As in prokaryotes, there is a very high proportion of coding DNA and an
absence of repeats. Mitochondrial genes are
[transcribed](transcription_(genetics) "wikilink") as multigenic
transcripts, which are cleaved and
[polyadenylated](Polyadenylation "wikilink") to yield mature
[mRNAs](mRNA "wikilink"). Most proteins necessary for mitochondrial
function are encoded by genes in the [cell
nucleus](cell_nucleus "wikilink") and the corresponding proteins are
imported into the mitochondrion.[215] The exact number of genes encoded
by the nucleus and the [mitochondrial
genome](Mitochondrial_DNA "wikilink") differs between species. Most
mitochondrial genomes are circular.[216] In general, mitochondrial DNA
lacks [introns](intron "wikilink"), as is the case in the human
mitochondrial genome;[217] however, introns have been observed in some
eukaryotic mitochondrial DNA,[218] such as that of
[yeast](yeast "wikilink")[219] and [protists](protist "wikilink"),[220]
including *[Dictyostelium](Dictyostelium "wikilink") discoideum*.[221]
Between protein-coding regions, tRNAs are present. Mitochondrial tRNA
genes have different sequences from the nuclear tRNAs, but lookalikes of
mitochondrial tRNAs have been found in the nuclear chromosomes with high
sequence similarity.[222]

In animals, the mitochondrial genome is typically a single circular
chromosome that is approximately 16 kb long and has 37 genes. The genes,
while highly conserved, may vary in location. Curiously, this pattern is
not found in the human body louse (*[Pediculus
humanus](Pediculus_humanus "wikilink")*). Instead, this mitochondrial
genome is arranged in 18 minicircular chromosomes, each of which is 3–4
kb long and has one to three genes.[223] This pattern is also found in
other [sucking lice](Anoplura "wikilink"), but not in [chewing
lice](Mallophaga "wikilink"). Recombination has been shown to occur
between the minichromosomes.

### Human population genetic studies

The near-absence of [genetic
recombination](genetic_recombination "wikilink") in mitochondrial DNA
makes it a useful source of information for studying [population
genetics](population_genetics "wikilink") and [evolutionary
biology](evolutionary_biology "wikilink").[224] Because all the
mitochondrial DNA is inherited as a single unit, or
[haplotype](haplotype "wikilink"), the relationships between
mitochondrial DNA from different individuals can be represented as a
[gene tree](phylogenetic_tree "wikilink"). Patterns in these gene trees
can be used to infer the evolutionary history of populations. The
classic example of this is in [human evolutionary
genetics](human_evolutionary_genetics "wikilink"), where the [molecular
clock](molecular_clock "wikilink") can be used to provide a recent date
for [mitochondrial Eve](mitochondrial_Eve "wikilink").[225][226] This is
often interpreted as strong support for a recent modern human expansion
[out of Africa](Recent_single-origin_hypothesis "wikilink").[227]
Another human example is the sequencing of mitochondrial DNA from
[Neanderthal](Neanderthal "wikilink") bones. The relatively large
evolutionary distance between the mitochondrial DNA sequences of
Neanderthals and living humans has been interpreted as evidence for the
lack of interbreeding between Neanderthals and modern humans.[228]

However, mitochondrial DNA reflects only the history of the females in a
population. This can be partially overcome by the use of paternal
genetic sequences, such as the
[non-recombining](genetic_recombination "wikilink") region of the
[Y-chromosome](Y-chromosome "wikilink").[229]

Recent measurements of the [molecular clock](molecular_clock "wikilink")
for mitochondrial DNA[230] reported a value of 1 mutation every 7884
years dating back to the most recent common ancestor of humans and apes,
which is consistent with estimates of mutation rates of autosomal DNA
(10 per base per generation).[231]

### Alternative genetic code

<div style="border: 1px solid grey; float:right; margin-right:0.6em; padding:0.5em">

<table>
<caption>Exceptions to the standard genetic code in mitochondria<a
href="#fn1" class="footnote-ref" id="fnref1"
role="doc-noteref"><sup>1</sup></a></caption>
<thead>
<tr class="header">
<th><p>Organism</p></th>
<th><p>Codon</p></th>
<th><p>Standard</p></th>
<th><p>Mitochondria</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><p><a href="Mammal" title="wikilink">Mammals</a></p></td>
<td><p>AGA, AGG</p></td>
<td><p>Arginine</p></td>
<td><p>Stop codon</p></td>
</tr>
<tr class="even">
<td><p><a href="Invertebrate"
title="wikilink">Invertebrates</a></p></td>
<td><p>AGA, AGG</p></td>
<td><p>Arginine</p></td>
<td><p>Serine</p></td>
</tr>
<tr class="odd">
<td><p><a href="Fungus" title="wikilink">Fungi</a></p></td>
<td><p>CUA</p></td>
<td><p>Leucine</p></td>
<td><p>Threonine</p></td>
</tr>
<tr class="even">
<td rowspan="2"><p>All of the above</p></td>
<td><p>AUA</p></td>
<td><p>Isoleucine</p></td>
<td><p>Methionine</p></td>
</tr>
<tr class="odd">
<td><p>UGA</p></td>
<td><p>Stop codon</p></td>
<td><p>Tryptophan</p></td>
</tr>
</tbody>
</table>
</div>

While slight variations on the standard [genetic
code](genetic_code "wikilink") had been predicted earlier,[233] none was
discovered until 1979, when researchers studying [human mitochondrial
genes](human_mitochondrial_genetics "wikilink") determined that they
used an alternative code.[234] Nonetheless, the mitochondria of many
other eukaryotes, including most plants, use the standard code.[235]
Many slight variants have been discovered since,[236] including various
alternative mitochondrial codes.[237] Further, the AUA, AUC, and AUU
codons are all allowable start codons.

Some of these differences should be regarded as pseudo-changes in the
genetic code due to the phenomenon of [RNA
editing](RNA_editing "wikilink"), which is common in mitochondria. In
higher plants, it was thought that CGG encoded for
[tryptophan](tryptophan "wikilink") and not
[arginine](arginine "wikilink"); however, the codon in the processed RNA
was discovered to be the UGG codon, consistent with the standard
[genetic code](genetic_code "wikilink") for tryptophan.[238] Of note,
the arthropod mitochondrial genetic code has undergone parallel
evolution within a phylum, with some organisms uniquely translating AGG
to lysine.[239]

### Replication and inheritance

Mitochondria divide by [mitochondrial
fission](mitochondrial_fission "wikilink"), a form of [binary
fission](binary_fission "wikilink") that is also done by bacteria[240]
although the process is tightly regulated by the host eukaryotic cell
and involves communication between and contact with several other
organelles. The regulation of this division differs between eukaryotes.
In many single-celled eukaryotes, their growth and division are linked
to the [cell cycle](cell_cycle "wikilink"). For example, a single
mitochondrion may divide synchronously with the nucleus. This division
and segregation process must be tightly controlled so that each daughter
cell receives at least one mitochondrion. In other eukaryotes (in
mammals for example), mitochondria may replicate their DNA and divide
mainly in response to the energy needs of the cell, rather than in phase
with the cell cycle. When the energy needs of a cell are high,
mitochondria grow and divide. When energy use is low, mitochondria are
destroyed or become inactive. In such examples mitochondria are
apparently randomly distributed to the daughter cells during the
division of the [cytoplasm](cytoplasm "wikilink"). Mitochondrial
dynamics, the balance between [mitochondrial
fusion](mitochondrial_fusion "wikilink") and
[fission](mitochondrial_fission "wikilink"), is an important factor in
pathologies associated with several disease conditions.[241]

The hypothesis of mitochondrial binary fission has relied on the
visualization by fluorescence microscopy and conventional [transmission
electron microscopy](transmission_electron_microscopy "wikilink") (TEM).
The resolution of fluorescence microscopy (~200 nm) is insufficient to
distinguish structural details, such as double mitochondrial membrane in
mitochondrial division or even to distinguish individual mitochondria
when several are close together. Conventional TEM has also some
technical limitations in verifying mitochondrial division.
[Cryo-electron tomography](Cryo-electron_tomography "wikilink") was
recently used to visualize mitochondrial division in frozen hydrated
intact cells. It revealed that mitochondria divide by budding.[242]

An individual's mitochondrial genes are inherited only from the mother,
with rare exceptions.[243] In humans, when an [egg
cell](ovum "wikilink") is fertilized by a sperm, the mitochondria, and
therefore the mitochondrial DNA, usually come from the egg only. The
sperm's mitochondria enter the egg, but do not contribute genetic
information to the embryo.[244] Instead, paternal mitochondria are
marked with [ubiquitin](ubiquitin "wikilink") to select them for later
destruction inside the [embryo](embryo "wikilink").[245] The egg cell
contains relatively few mitochondria, but these mitochondria divide to
populate the cells of the adult organism. This mode is seen in most
organisms, including the majority of animals. However, mitochondria in
some species can sometimes be inherited paternally. This is the norm
among certain [coniferous](conifer "wikilink") plants, although not in
[pine trees](pine_tree "wikilink") and [yews](taxus "wikilink").[246]
For [Mytilids](Mytilidae "wikilink") , paternal inheritance only occurs
within males of the species.[247][248][249] It has been suggested that
it occurs at a very low level in humans.[250]

[Uniparental inheritance](Uniparental_inheritance "wikilink") leads to
little opportunity for [genetic
recombination](genetic_recombination "wikilink") between different
lineages of mitochondria, although a single mitochondrion can contain
2–10 copies of its DNA.[251] What recombination does take place
maintains genetic integrity rather than maintaining diversity. However,
there are studies showing evidence of recombination in mitochondrial
DNA. It is clear that the enzymes necessary for recombination are
present in mammalian cells.[252] Further, evidence suggests that animal
mitochondria can undergo recombination.[253] The data are more
controversial in humans, although indirect evidence of recombination
exists.[254][255]

Entities undergoing uniparental inheritance and with little to no
recombination may be expected to be subject to [Muller's
ratchet](Muller's_ratchet "wikilink"), the accumulation of deleterious
mutations until functionality is lost. Animal populations of
mitochondria avoid this buildup through a developmental process known as
the [mtDNA
bottleneck](Heteroplasmy#Mitochondrial_bottleneck "wikilink"). The
bottleneck exploits [stochastic processes in the
cell](cellular_noise "wikilink") to increase the cell-to-cell
variability in [mutant load](heteroplasmy "wikilink") as an organism
develops: a single egg cell with some proportion of mutant mtDNA thus
produces an embryo where different cells have different mutant loads.
Cell-level selection may then act to remove those cells with more mutant
mtDNA, leading to a stabilization or reduction in mutant load between
generations. The mechanism underlying the bottleneck is
debated,[256][257][258] with a recent mathematical and experimental
metastudy providing evidence for a combination of random partitioning of
mtDNAs at cell divisions and random turnover of mtDNA molecules within
the cell.[259]

### DNA repair

Mitochondria can repair oxidative [DNA
damage](DNA_damage_(naturally_occurring) "wikilink") by mechanisms
analogous to those occurring in the [cell
nucleus](cell_nucleus "wikilink"). The proteins employed in
[mtDNA](mitochondrial_DNA "wikilink") repair are encoded by nuclear
[genes](gene "wikilink"), and are translocated to the mitochondria. The
[DNA repair](DNA_repair "wikilink") pathways in mammalian mitochondria
include [base excision repair](base_excision_repair "wikilink"),
double-strand break repair, direct reversal and [mismatch
repair](DNA_mismatch_repair "wikilink").[260][261] Alternatively, DNA
damage may be bypassed, rather than repaired, by translesion synthesis.

Of the several DNA repair process in mitochondria, the base excision
repair pathway has been most comprehensively studied.[262] Base excision
repair is carried out by a sequence of enzyme-catalyzed steps that
include recognition and excision of a damaged DNA base, removal of the
resulting abasic site, end processing, gap filling and ligation. A
common damage in mtDNA that is repaired by base excision repair is
[8-oxoguanine](8-oxoguanine "wikilink") produced by oxidation of
[guanine](guanine "wikilink").[263]

Double-strand breaks can be repaired by [homologous
recombinational](homologous_recombination "wikilink") repair in both
mammalian mtDNA[264] and plant mtDNA.[265] Double-strand breaks in mtDNA
can also be repaired by [microhomology-mediated end
joining](microhomology-mediated_end_joining "wikilink").[266] Although
there is evidence for the repair processes of direct reversal and
mismatch repair in mtDNA, these processes are not well
characterized.[267]

### Lack of mitochondrial DNA

Some organisms have lost mitochondrial DNA altogether. In these cases,
genes encoded by the mitochondrial DNA have been lost or transferred to
the nucleus.[268] *[Cryptosporidium](Cryptosporidium "wikilink")* have
mitochondria that lack any DNA, presumably because all their genes have
been lost or transferred.[269] In *Cryptosporidium*, the mitochondria
have an altered [ATP](Adenosine_triphosphate "wikilink") generation
system that renders the parasite resistant to many classical
mitochondrial [inhibitors](enzyme_inhibitor "wikilink") such as
[cyanide](cyanide "wikilink"), [azide](azide "wikilink"), and
[atovaquone](atovaquone "wikilink").[270] Mitochondria that lack their
own DNA have been found in a marine parasitic
[dinoflagellate](dinoflagellate "wikilink") from the genus
*[Amoebophyra](Amoebophyra "wikilink")*. This microorganism, *A.
cerati*, has functional mitochondria that lack a genome.[271] In related
species, the mitochondrial genome still has three genes, but in *A.
cerati* only a single mitochondrial gene — the [cytochrome c oxidase
I](Cytochrome_c_oxidase_subunit_I "wikilink") gene (*cox1*) — is found,
and it has migrated to the genome of the nucleus.[272]

## Dysfunction and disease

### Mitochondrial diseases

Damage and subsequent dysfunction in mitochondria is an important factor
in a range of human diseases due to their influence in cell metabolism.
Mitochondrial disorders often present as neurological disorders,
including [autism](autism "wikilink").[273] They can also manifest as
[myopathy](myopathy "wikilink"), [diabetes](diabetes "wikilink"),
multiple [endocrinopathy](endocrinopathy "wikilink"), and a variety of
other systemic disorders.[274] Diseases caused by mutation in the mtDNA
include [Kearns–Sayre syndrome](Kearns–Sayre_syndrome "wikilink"),
[MELAS syndrome](MELAS_syndrome "wikilink") and [Leber's hereditary
optic neuropathy](Leber's_hereditary_optic_neuropathy "wikilink").[275]
In the vast majority of cases, these diseases are transmitted by a
female to her children, as the [zygote](zygote "wikilink") derives its
mitochondria and hence its mtDNA from the ovum. Diseases such as
Kearns-Sayre syndrome, [Pearson syndrome](Pearson_syndrome "wikilink"),
and progressive external [ophthalmoplegia](ophthalmoparesis "wikilink")
are thought to be due to large-scale mtDNA rearrangements, whereas other
diseases such as MELAS syndrome, Leber's hereditary optic neuropathy,
[MERRF syndrome](MERRF_syndrome "wikilink"), and others are due to
[point mutations](point_mutation "wikilink") in mtDNA.[276]

It has also been reported that drug tolerant cancer cells have an
increased number and size of mitochondria which suggested an increase in
mitochondrial biogenesis.[277] A 2022 study in *Nature Nanotechnology*
has reported that cancer cells can hijack the mitochondria from immune
cells via physical tunneling nanotubes.[278]

In other diseases, defects in nuclear genes lead to dysfunction of
mitochondrial proteins. This is the case in [Friedreich's
ataxia](Friedreich's_ataxia "wikilink"), [hereditary spastic
paraplegia](hereditary_spastic_paraplegia "wikilink"), and [Wilson's
disease](Wilson's_disease "wikilink").[279] These diseases are inherited
in a [dominance relationship](dominance_relationship "wikilink"), as
applies to most other genetic diseases. A variety of disorders can be
caused by nuclear mutations of oxidative phosphorylation enzymes, such
as [coenzyme Q10](coenzyme_Q10 "wikilink") deficiency and [Barth
syndrome](Barth_syndrome "wikilink").[280] Environmental influences may
interact with hereditary predispositions and cause mitochondrial
disease. For example, there may be a link between
[pesticide](pesticide "wikilink") exposure and the later onset of
[Parkinson's disease](Parkinson's_disease "wikilink").[281][282] Other
pathologies with etiology involving mitochondrial dysfunction include
[schizophrenia](schizophrenia "wikilink"), [bipolar
disorder](bipolar_disorder "wikilink"), [dementia](dementia "wikilink"),
[Alzheimer's disease](Alzheimer's_disease "wikilink"),[283][284]
Parkinson's disease, [epilepsy](epilepsy "wikilink"),
[stroke](stroke "wikilink"), [cardiovascular
disease](cardiovascular_disease "wikilink"), [chronic fatigue
syndrome](chronic_fatigue_syndrome "wikilink"), [retinitis
pigmentosa](retinitis_pigmentosa "wikilink"), and [diabetes
mellitus](diabetes_mellitus "wikilink").[285][286]

Mitochondria-mediated oxidative stress plays a role in cardiomyopathy in
[type 2 diabetics](type_2_diabetics "wikilink"). Increased fatty acid
delivery to the heart increases fatty acid uptake by cardiomyocytes,
resulting in increased fatty acid oxidation in these cells. This process
increases the reducing equivalents available to the electron transport
chain of the mitochondria, ultimately increasing reactive oxygen species
(ROS) production. ROS increases [uncoupling
proteins](uncoupling_proteins "wikilink") (UCPs) and potentiate proton
leakage through the [adenine nucleotide
translocator](adenine_nucleotide_translocator "wikilink") (ANT), the
combination of which [uncouples](uncoupler "wikilink") the mitochondria.
Uncoupling then increases oxygen consumption by the mitochondria,
compounding the increase in fatty acid oxidation. This creates a vicious
cycle of uncoupling; furthermore, even though oxygen consumption
increases, ATP synthesis does not increase proportionally because the
mitochondria are uncoupled. Less ATP availability ultimately results in
an energy deficit presenting as reduced cardiac efficiency and
contractile dysfunction. To compound the problem, impaired sarcoplasmic
reticulum calcium release and reduced mitochondrial reuptake limits peak
cytosolic levels of the important signaling ion during muscle
contraction. Decreased intra-mitochondrial calcium concentration
increases dehydrogenase activation and ATP synthesis. So in addition to
lower ATP synthesis due to fatty acid oxidation, ATP synthesis is
impaired by poor calcium signaling as well, causing cardiac problems for
diabetics.[287]

Mitochondria also modulate processes such as testicular somatic cell
development, spermatogonial stem cell differentiation, luminal
acidification, testosterone production in testes, and more. Thus,
dysfunction of mitochondria in spermatozoa can be a cause for
infertility.[288]

In efforts to combat mitochondrial disease, [mitochondrial replacement
therapy](mitochondrial_replacement_therapy "wikilink") (MRT) has been
developed. This form of in vitro fertilization uses donor mitochondria,
which avoids the transmission of diseases caused by mutations of
mitochondrial DNA.[289] However, this therapy is still being researched
and can introduce genetic modification, as well as safety concerns.
These diseases are rare but can be extremely debilitating and
progressive diseases, thus posing complex ethical questions for public
policy.[290]

### Relationships to aging

There may be some leakage of the [electrons
transferred](Electron_transport_chain "wikilink") in the respiratory
chain to form [reactive oxygen
species](reactive_oxygen_species "wikilink"). This was thought to result
in significant [oxidative stress](oxidative_stress "wikilink") in the
mitochondria with high mutation rates of mitochondrial DNA.[291]
Hypothesized links between aging and oxidative stress are not new and
were proposed in 1956,[292] which was later refined into the
[mitochondrial free radical theory of
aging](Mitochondrial_theory_of_ageing "wikilink").[293] A vicious cycle
was thought to occur, as oxidative stress leads to mitochondrial DNA
mutations, which can lead to enzymatic abnormalities and further
oxidative stress.

A number of changes can occur to mitochondria during the aging
process.[294] Tissues from elderly humans show a decrease in enzymatic
activity of the proteins of the respiratory chain.[295] However, mutated
mtDNA can only be found in about 0.2% of very old cells.[296] Large
deletions in the mitochondrial genome have been hypothesized to lead to
high levels of [oxidative stress](oxidative_stress "wikilink") and
neuronal death in [Parkinson's
disease](Parkinson's_disease "wikilink").[297] Mitochondrial dysfunction
has also been shown to occur in [amyotrophic lateral
sclerosis](amyotrophic_lateral_sclerosis "wikilink").[298][299]

Since mitochondria cover a pivotal role in the ovarian function, by
providing ATP necessary for the development from germinal vesicle to
mature [oocyte](oocyte "wikilink"), a decreased mitochondria function
can lead to inflammation, resulting in premature ovarian failure and
accelerated ovarian aging. The resulting dysfunction is then reflected
in quantitative (such as mtDNA copy number and mtDNA deletions),
qualitative (such as mutations and strand breaks) and oxidative damage
(such as dysfunctional mitochondria due to ROS), which are not only
relevant in ovarian aging, but perturb oocyte-cumulus crosstalk in the
ovary, are linked to genetic disorders (such as Fragile X) and can
interfere with embryo selection.[300]

## History

The first observations of intracellular structures that probably
represented mitochondria were published in 1857, by the physiologist
[Albert von Kolliker](Albert_von_Kölliker "wikilink").[301][302]
[Richard Altmann](Richard_Altmann "wikilink"), in 1890, established them
as cell organelles and called them "bioblasts."[303][304] In 1898, Carl
Benda coined the term "mitochondria" from the
[Greek](Greek_language "wikilink") , , "thread", and , ,
"granule."[305][306]<ref> From p. 397: After Brenda states that *" ...
ich bereits in vielen Zellarten aller möglichen Thierclassen gefunden
habe, ... "* ( ... I have already found \[them (mitochondria)\] in many
types of cells of all possible classes of animals, ... ), he suggests:
*"Ich möchte vorläufig vorschlagen, ihnen als Mitochondria eine
besondere Stellung vorzubehalten, die ich in weiteren Arbeiten begründen
werde."* (I would like to suggest provisionally reserving for them, as
"mitochondria", a special status which I will justify in further work.)

-   </ref>

    [Leonor Michaelis](Leonor_Michaelis "wikilink") discovered that
    [Janus green](Janus_green "wikilink") can be used as a [supravital
    stain](supravital_stain "wikilink") for mitochondria in 1900.[307]
    In 1904, [Friedrich Meves](Friedrich_Meves "wikilink") made the
    first recorded observation of mitochondria in plants in cells of the
    white waterlily, *[Nymphaea
    alba](Nymphaea_alba "wikilink"),*[308][309] and in 1908, along with
    [Claudius Regaud](Claudius_Regaud "wikilink"), suggested that they
    contain proteins and lipids. Benjamin F. Kingsbury, in 1912, first
    related them with cell respiration, but almost exclusively based on
    morphological observations.[310][311] In 1913, [Otto Heinrich
    Warburg](Otto_Heinrich_Warburg "wikilink") linked respiration to
    particles which he had obtained from extracts of guinea-pig liver
    and which he called "grana".[312] Warburg and [Heinrich Otto
    Wieland](Heinrich_Otto_Wieland "wikilink"), who had also postulated
    a similar particle mechanism, disagreed on the chemical nature of
    the respiration. It was not until 1925, when [David
    Keilin](David_Keilin "wikilink") discovered
    [cytochromes](cytochromes "wikilink"), that the [respiratory
    chain](respiratory_chain "wikilink") was described.[313]

In 1939, experiments using minced muscle cells demonstrated that
cellular respiration using one [oxygen molecule](Oxygen "wikilink") can
form four [adenosine triphosphate](adenosine_triphosphate "wikilink")
(ATP) molecules, and in 1941, the concept of the phosphate bonds of ATP
being a form of energy in cellular metabolism was developed by [Fritz
Albert Lipmann](Fritz_Albert_Lipmann "wikilink"). In the following
years, the mechanism behind cellular respiration was further elaborated,
although its link to the mitochondria was not known.[314] The
introduction of [tissue fractionation](Cell_fractionation "wikilink") by
[Albert Claude](Albert_Claude "wikilink") allowed mitochondria to be
isolated from other cell fractions and biochemical analysis to be
conducted on them alone. In 1946, he concluded that [cytochrome
oxidase](cytochrome_oxidase "wikilink") and other enzymes responsible
for the respiratory chain were isolated to the mitochondria. [Eugene
Kennedy](Eugene_P._Kennedy "wikilink") and [Albert
Lehninger](Albert_L._Lehninger "wikilink") discovered in 1948 that
mitochondria are the site of [oxidative
phosphorylation](oxidative_phosphorylation "wikilink") in eukaryotes.
Over time, the fractionation method was further developed, improving the
quality of the mitochondria isolated, and other elements of [cell
respiration](Cellular_respiration "wikilink") were determined to occur
in the mitochondria.[315]

The first high-resolution electron [micrographs](micrographs "wikilink")
appeared in 1952, replacing the Janus Green stains as the preferred way
to visualize mitochondria.[316] This led to a more detailed analysis of
the structure of the mitochondria, including confirmation that they were
surrounded by a membrane. It also showed a second membrane inside the
mitochondria that folded up in ridges dividing up the inner chamber and
that the size and shape of the mitochondria varied from cell to cell.

The popular term "powerhouse of the cell" was coined by [Philip
Siekevitz](Philip_Siekevitz "wikilink") in 1957.[317][318]

In 1967, it was discovered that mitochondria contained
[ribosomes](ribosomes "wikilink").[319] In 1968, methods were developed
for mapping the mitochondrial genes, with the genetic and physical map
of yeast mitochondrial DNA completed in 1976.[320]

## See also

-   [Anti-mitochondrial
    antibodies](Anti-mitochondrial_antibodies "wikilink")
-   [Mitochondrial metabolic
    rates](Respirometry#Mitochondrial_metabolic_rates "wikilink")
-   [Mitochondrial permeability transition
    pore](Mitochondrial_permeability_transition_pore "wikilink")
-   [Mitophagy](Mitophagy "wikilink")
-   [Nebenkern](Nebenkern "wikilink")
-   [Oncocyte](Oncocyte "wikilink")
-   [Oncocytoma](Oncocytoma "wikilink")
-   [Paternal mtDNA
    transmission](Paternal_mtDNA_transmission "wikilink")
-   [Plastid](Plastid "wikilink")
-   [Submitochondrial particle](Submitochondrial_particle "wikilink")
